_drafts/release-workflows.md

```diff
@@ -0,0 +1,388 @@
+
+That enforces consistence across versions: **if a `x.y.z` version exists on a given channel, all superior versions on that channel must includes all the commits of `x.y.z`**
+
+If not specified the `type` is `major`.
+
+With specific channel names:
+
+```json
+"release": {
+  "branches": [
+    "master", 
+    {
+      "branch": "next",
+      "channel": "experimental",
+      "type": "major"
+    }
+  ]
+}
+```
+
+#### LTS branches
+
+Workflow to release legacy versions.
+
+Ideal for modules maintaining legacy versions and doing bug fixes and features only releases.
+
+Branch: `1.x.x`, `2.x.x`, `master` (any number of lts branches are supported)
+
+Channel: `latest` if supported by the release target
+
+##### LTS branches workflow
+
+| Action                                              | Result                                                                                                           |
+|-----------------------------------------------------|------------------------------------------------------------------------------------------------------------------|
+| **push** to `master`                                | Release on default channel (if supported, otherwise no channel)                                                  |
+| **merge** feature branch to `master`                | Release on default channel (if supported, otherwise no channel)                                                  |
+| **push** to `next`                                  | -                                                                                                                |
+| **push** to `1.x.x` or `2.x.x` branches             | Release on default channel (if supported, otherwise no channel) if in range, report an error and error otherwise |
+| **merge** any branch to `1.x.x` or `2.x.x` branches | Release on default channel (if supported, otherwise no channel) if in range, report an error and error otherwise |
+
+Note: By default lts releases are published on the default channel as they are limited to a range of versions inferior to the versions released from `master`. Dependents will get only the expected releases by specifying a range dependency (for example `^2.0.0`).
+If a `channel` is specified, then the release will be made on this channel.
+
+##### LTS branches configuration
+
+```json
+"release": {
+  "branches": ["1.x.x", "2.x.x", "master"]
+  }
+}
+```
+
+With specific channel names:
+
+```json
+"release": {
+  "branches": [{
+    "branch": "v1",
+    "range": "1.x.x",
+    "channel": "v1"
+  }, {
+    "branch": "v2",
+    "range": "2.x.x",
+    "channel": "v2"
+  }, "master"]
+}
+```
+
+#### Pre-releases
+
+Workflow to distribute releases without incrementing the semantic version during the development of new a application version.  
+Ideal for applications distributing unstable/alpha/preview releases.
+
+Branch: `master`, `next-beta`, `next-alpha`
+
+Channel: `latest` if supported by the release target
```

Comment:
as mentioned in https://github.com/semantic-release/semantic-release/issues/563#issuecomment-357488470, it might help to call out that pre-releases are published to non-`latest` channels and that the preid will be used if the channel is not specified in the config

Reply:
Absolutly